server/accounts.go

```diff
@@ -161,6 +163,10 @@ const (
 	Chunked
 )
 
+var commaSeparatorRegEx = regexp.MustCompile(`,\s*`)
+var partitionMappingFunctionRegEx = regexp.MustCompile(`{{partition *\((.*)\)}}`)
+var wildcardMappingFunctionRegEx = regexp.MustCompile(`{{wildcard *\((.*)\)}}`)
```

Comment:
maybe `{{\s*wildcard\s*\((.*)\)\s*}}` to allow `{{ wildcard(foo) }}` also, that seems more likely than `{{wildcard (foo)}}` but since we support one we might as well do both ways allowing space? People who used moustache things will expect that to be allowed

Reply:
Good suggestion, implemented by the latest commit